### PR TITLE
Frontend enhanced

### PR DESCRIPTION
**Pull request by marioolislaegers**

### CODE REVIEW

**Review by SvanSomeren — CHANGES_REQUESTED** [review]
remove one of the two enhancements.

File: src/assets/scss/style.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
 @import "bulma/sass/elements/button.sass";
 @import "bulma/sass/elements/content.sass";
 @import "bulma/sass/elements/container.sass";
+@import "bulma/sass/elements/tag.sass";
```
Comment (tomdewildt) [inline src/assets/scss/style.scss]: Remove these changes

File: src/components/table/TableCell.js [hunk 2]
```diff
@@ -7,6 +8,8 @@ import {
     TableCellBoolean,
     TableCellDate,
     TableCellContextMenu,
+    TableCellList,
+    TableCellIcon,
```
Comment (tomdewildt) [inline src/components/table/TableCell.js:10]: Idem

File: src/components/table/TableCell.js [hunk 3]
```diff
@@ -24,6 +27,17 @@ const TableCell = ( {
                     value={ value }
                     link={ link }
                 />
+            ); case CELL_TYPES.ICON:
+            return (
+                <TableCellIcon
+                    value={ value }
+                />
+            );
+        case CELL_TYPES.LIST:
```
Comment (tomdewildt) [inline src/components/table/TableCell.js]: Idem

File: src/components/table/cells/List.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+const generateKey = pre => `${ pre }_${ new Date().getTime() }`;
+const TableCellList = ( { value } ) => (
+    <td>
+        <div className="tags">
+
+            {
+                value.split( "," ).map( v => (
+                    <span key={ generateKey( "tags" ) } className="tag is-info  ">{v.trim()}</span> ) )
+            }
+
```
Comment (tomdewildt) [inline src/components/table/cells/List.js]: Remove these whitespaces

File: src/components/table/cells/List.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const generateKey = pre => `${ pre }_${ new Date().getTime() }`;
```
Comment (tomdewildt) [inline src/components/table/cells/List.js]: Use a value as key

File: src/components/table/cells/index.js [hunk 6]
```diff
@@ -4,3 +4,5 @@ export { default as TableCellFilter } from "./Filter";
 export { default as TableCellSort } from "./Sort";
 export { default as TableCellText } from "./Text";
 export { default as TableCellDate } from "./Date";
+export { default as TableCellList } from "./List";
```
Comment (tomdewildt) [inline src/components/table/cells/index.js]: Idem

File: src/tables/Personnel.js [hunk 7]
```diff
@@ -20,7 +20,7 @@ export default [
     {
         label: "Roles",
         key: "roles",
-        type: CELL_TYPES.TEXT,
+        type: CELL_TYPES.LIST,
```
Comment (tomdewildt) [inline src/tables/Personnel.js]: Idem

File: src/tables/Types.js [hunk 8]
```diff
@@ -3,6 +3,7 @@ const CELL_TYPES = {
     DATE: "date",
     TIME: "time",
     BOOLEAN: "boolean",
+    LIST: "list",
```
Comment (tomdewildt) [inline src/tables/Types.js]: Idem

**Review by tomdewildt — CHANGES_REQUESTED** [review]

**Pull request closed without merging**